sdk/include/psdk/shellapi.h

```diff
@@ -46,8 +46,10 @@ extern "C" {
  * of discovering them. If somebody else can verify them, it would be great.
  */
 #define SEE_MASK_HASLINKNAME	0x00010000
-#define SEE_MASK_HASTITLE	0x00020000
-#define SEE_MASK_FLAG_SEPVDM	0x00040000
+#define SEE_MASK_HASTITLE       0x00080000
+#define SEE_MASK_FLAG_SEPVDM    0x00020000
+#define SEE_MASK_UNKNOWN_0x1000 0x00001000 /* FIXME: Name */
+#define SEE_MASK_USE_RESERVED   0x00040000
```

Comment:
Can you reorder these five defines by value?

Reply:
Fixed.